src/test/groovy/com/github/vkorobkov/jfixturescmd/PromptTest.groovy

```diff
@@ -18,7 +18,7 @@ class PromptTest extends Specification {
         when:
         write()
         then:
-        2 * printStream.write(_)
-        2 * printStream.flush()
+        4 * printStream.write(_)
```

Comment:
Can we remove this counter at all?
Just `printStream.write(_)` and `printStream.flush()` should work